test/OpenTelemetry.Exporter.Geneva.Tests/OtlpProtobufMetricExporterTests.cs

```diff
@@ -187,6 +292,50 @@ public void CounterSerializationSingleMetricPoint(string instrumentName, long? l
             Assert.Equal(doubleValue, dataPoint.AsDouble);
         }
 
+#if EXPOSE_EXPERIMENTAL_FEATURES
+        if (isExemplarsEnabled)
+        {
+            Assert.Single(dataPoint.Exemplars);
+
+            var exemplar = dataPoint.Exemplars[0];
+
+            Assert.NotEqual(default, exemplar.TimeUnixNano);
```

Comment:
We could have a stricter assertion by ensuring that `exemplar.TimeUnixNanon` lies within an acceptable range. Something like this:
```csharp
var start = DateTime.UtcNow; //convert to unix nanoseconds
counter.Add(longValue.Value, this.exemplarTagList);
var end = DateTime.UtcNow;
...
Assert.True(exemplar.TimeUnixNano> start && exemplar.TimeUnixNano < end);
```

Reply:
Or maybe compare it with the exact timestamp by inspecting the Exemplars in the MetricPoint.

Reply:
Added